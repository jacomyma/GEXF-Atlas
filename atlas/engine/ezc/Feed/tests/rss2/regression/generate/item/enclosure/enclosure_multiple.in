<?php
return array( 'title' => 'Feed title',
              'link'  => array( array( 'href' => 'Feed link' ) ),
              'description' => 'Feed description',
			  'item' => array( array( 'title' => 'Item title 1',
                                      'link' => 'Item link 1',
                                      'description' => 'Item description 1',
                                      'enclosure' => array( 'url' => 'Enclosure url 1',
                                                            'length' => 1200,
                                                            'type' => 'Enclosure type 1' ) ),
                               array( 'title' => 'Item title 2',
                                      'link' => 'Item link 2',
                                      'description' => 'Item description 2',
                                      'enclosure' => array( 'url' => 'Enclosure url 2',
                                                            'length' => 2400,
                                                            'type' => 'Enclosure type 2' ) ), ),
            );
?>
