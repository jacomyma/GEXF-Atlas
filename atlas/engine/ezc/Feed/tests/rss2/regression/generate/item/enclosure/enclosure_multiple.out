<?xml version="1.0" encoding="utf-8"?>
<rss version="2.0">
  <channel>
    <title>Feed title</title>
    <link>Feed link</link>
    <description>Feed description</description>
    <pubDate>XXX</pubDate>
    <generator>eZ Components</generator>
    <docs>http://www.rssboard.org/rss-specification</docs>
    <item>
      <title>Item title 1</title>
      <link>Item link 1</link>
      <description>Item description 1</description>
      <enclosure url="Enclosure url 1" length="1200" type="Enclosure type 1"/>
    </item>
    <item>
      <title>Item title 2</title>
      <link>Item link 2</link>
      <description>Item description 2</description>
      <enclosure url="Enclosure url 2" length="2400" type="Enclosure type 2"/>
    </item>
  </channel>
</rss>
